src/ucp/core/ucp_rkey.c

```diff
@@ -588,7 +588,7 @@ ucp_lane_index_t ucp_rkey_find_rma_lane(ucp_context_h context,
     uint8_t rkey_index;
     int prio;
 
-    for (prio = 0; prio < UCP_MAX_LANES; ++prio) {
+    for (prio = 0; prio < UCP_MAX_FAST_PATH_LANES; ++prio) {
```

Comment:
ucs_static_array_size

Reply:
how? we don't have the size of the array

Reply:
maybe pass it as arg?
since we pass lanes array as arg